src/components/common/Stepper.vue

```diff
@@ -26,7 +26,7 @@
             mdi-check-circle
           </v-icon>
           <v-icon class="step__btn2" size="30" color="error darken-1"
-            v-show="!isValid(step.to) && getShowErrors && isReviewStepValid(step.to)">
+            v-show="!isValid(step.to) && getShowErrors && isReviewStepValid(step.to) && getValidateSteps">
```

Comment:
I know I'm picking on existing code but I'm trying to understand the logic here (validate it in my head) and I'm still confused by it.

You check `getValidateSteps` here and it's also checked in `isReviewStepValid()`...

I also notice that this code displays the Close icon on error, but one of the conditions is that `isReviewStepValid` is True (ie, valid)!?!?

Can you fix this as it's really hard to understand ... I suggest renaming isReviewStepValid -> isReviewStepInvalid, or adding a ! in front of it and refactoring the method to return the opposite of what it does now.

It may turn out that `getValidateSteps` doesn't need to be used twice 🤷

Reply:
I don't quite understand `isReviewStepValid()` either. It returns `route !== RouteNames.DISSOLUTION_REVIEW_CONFIRM || this.getValidateSteps`. This `v-icon` will be not displayed if `route === RouteNames.DISSOLUTION_REVIEW_CONFIRM` and `this.getValidateSteps == false`? It's really confusing.

The one that matters the most is `getValidateSteps`. The icon only displays when `getValidateSteps == true`, along with `!isValid(step.to) && getShowErrors`. I am thinking to remove `isReviewStepValid()`.

Reply:
> I am thinking to remove `isReviewStepValid()`.

The team that implemented dissolutions must have done this. It's not exactly clear to me why, but I do know that dissolutions are a single step, so maybe this is to handle that case. Best bet: check with @seeker25 or @meganawong on this.

I'd like to see whole statement simplified if possible (or explained if not 😃 ).

Reply:
This is done in this [pull request](https://github.com/bcgov/business-create-ui/pull/418).

Reply:
I've removed the method in my latest commit. I can add the method back if necessary. I've tested all scenarios list in the ticket and they all worked well.

Reply:
Oh, Alex did that, as you mentioned previously.

Please figure this out and either simplify it, or add comments explaining it.

Reply:
Alex is not here until next week. I can ask him why he implemented this when he is back.

Reply:
Can you tell from Alex's ticket description, or ask Alex's ticket author?

It actually seems that your ticket pre-empts Alex's ticket, though they were both written by Yui.